src/pages/Encounters/EncounterList.tsx

```diff
@@ -36,6 +36,55 @@ import query from "@/Utils/request/query";
 import { PaginatedResponse } from "@/Utils/request/types";
 import { Encounter, EncounterPriority } from "@/types/emr/encounter";
 
+const ENCOUNTER_PRIORITY = {
```

Comment:
check encounter.ts. you should be able to reuse them (tweak if need be).

Should be able to combine this and the emote one.